ffi/derive/tests/ui_fail/nested_owned_structure.rs

```diff
@@ -0,0 +1,21 @@
+use std::{alloc::alloc, mem::MaybeUninit};
```

Comment:
Is't `.stderr` file is missing?

Reply:
could be, I haven't fixed compile tests yet